services-js/commissions-app/src/server/services/Email.ts

```diff
@@ -0,0 +1,109 @@
+import { Client as PostmarkClient } from 'postmark';
+import Rollbar from 'rollbar';
+
+import EmailContent, {
+  TemplateData as ApplicationData,
+} from '../email/EmailContent';
+
+// see services-js/registry-certs/server/services/Emails.ts
+
+/**
+ * Sends a confirmation email to the applicant, as well as to the policy office.
+ */
+export default class Email {
+  from: string;
+  postmarkClient: PostmarkClient;
+  rollbar: Rollbar;
+  emailContent: EmailContent;
+
+  applicantSubject = 'We’ve received your application';
+  policyOfficeSubject = 'New City Clerk Boards and Commissions Application';
+  policyOfficeToAddress = process.env.POLICY_OFFICE_TO_ADDRESS ||
+  'boardsandcommissions@boston.gov';
+
+  constructor(from: string, postmarkClient: PostmarkClient, rollbar: Rollbar) {
+    this.from = from;
+    this.postmarkClient = postmarkClient;
+    this.rollbar = rollbar;
+
+    this.emailContent = new EmailContent();
+  }
+
+  // todo: need newId from db server
+  public sendConfirmations(validForm, fetchedBoards, applicationId) {
+    const applicationData: ApplicationData = {
+      commissionNames: namesFromIds(fetchedBoards, validForm.commissionIds),
+      firstName: validForm.firstName,
+      lastName: validForm.lastName,
+      email: validForm.email,
+      applicationId,
+    };
+
+    const renderedApplicantBodies = this.emailContent.renderApplicantBodies(
+      applicationData
+    );
+    const renderedPolicyOfficeBodies = this.emailContent.renderPolicyOfficeBodies(
+      applicationData
+    );
+
+    // send to applicant...
+    this.sendConfirmationEmail(
+      applicationData.email,
+      this.applicantSubject,
+      renderedApplicantBodies
+    ).catch(err => this.rollbar.error(`Error sending applicant email: ${err}`));
+
+    // send to policy office...
+    this.sendConfirmationEmail(
+      this.policyOfficeToAddress,
+      this.policyOfficeSubject,
+      renderedPolicyOfficeBodies
+    ).catch(err =>
+      this.rollbar.error(`Error sending policy office email: ${err}`)
+    );
+  }
+
+  /**
+   * Use Postmark to send an email. The user does not need to be notified
+   * if there is an error.
+   */
+  private sendConfirmationEmail(
+    emailTo: string,
+    subject: string,
+    renderedBodies
+  ): Promise<void> {
+    return new Promise((resolve, reject) => {
+      PostmarkClient.sendEmail(
+        {
+          To: emailTo,
+          From: this.from,
+          Subject: subject,
+          HtmlBody: renderedBodies.html,
+          TextBody: renderedBodies.text,
+        },
+        (err, result) => (err ? reject(err) : resolve(result))
+      );
+    });
+  }
+}
+
+/**
+ * Convert array of applied-for commissions into a human-readable list of
+ * names, and sort it alphabetically.
+ *
+ * @param {Commission[]} commissionsList
+ * @param {string[]} commissionIds
+ */
+export function namesFromIds(commissionsList, commissionIds) {
+  return commissionIds
+    .reduce((acc, id) => {
```

Comment:
This could also just be a `map` since you don’t need to do anything interesting with the accumulator.